model/config.go

```diff
@@ -2068,55 +2068,55 @@ type ClientRequirements struct {
-	GuestFilter       *string `access:"authentication"`
+	UserFilter        *string `access:"authentication_ldap"` // telemetry: none
+	GroupFilter       *string `access:"authentication_ldap"`
+	GuestFilter       *string `access:"authentication_ldap"`
 	EnableAdminFilter *bool
 	AdminFilter       *string
 
 	// Group Mapping
-	GroupDisplayNameAttribute *string `access:"authentication"`
-	GroupIdAttribute          *string `access:"authentication"`
+	GroupDisplayNameAttribute *string `access:"authentication_ldap"`
+	GroupIdAttribute          *string `access:"authentication_ldap"`
 
 	// User Mapping
-	FirstNameAttribute *string `access:"authentication"`
-	LastNameAttribute  *string `access:"authentication"`
-	EmailAttribute     *string `access:"authentication"`
-	UsernameAttribute  *string `access:"authentication"`
-	NicknameAttribute  *string `access:"authentication"`
-	IdAttribute        *string `access:"authentication"`
-	PositionAttribute  *string `access:"authentication"`
-	LoginIdAttribute   *string `access:"authentication"`
-	PictureAttribute   *string `access:"authentication"`
+	FirstNameAttribute *string `access:"authentication_ldap"`
+	LastNameAttribute  *string `access:"authentication_ldap"`
+	EmailAttribute     *string `access:"authentication_ldap"`
+	UsernameAttribute  *string `access:"authentication_ldap"`
+	NicknameAttribute  *string `access:"authentication_ldap"`
+	IdAttribute        *string `access:"authentication_ldap"`
+	PositionAttribute  *string `access:"authentication_ldap"`
+	LoginIdAttribute   *string `access:"authentication_ldap"`
+	PictureAttribute   *string `access:"authentication_ldap"`
 
 	// Synchronization
-	SyncIntervalMinutes *int `access:"authentication"`
+	SyncIntervalMinutes *int `access:"authentication_ldap"`
 
 	// Advanced
-	SkipCertificateVerification *bool   `access:"authentication"`
-	PublicCertificateFile       *string `access:"authentication"`
-	PrivateKeyFile              *string `access:"authentication"`
-	QueryTimeout                *int    `access:"authentication"`
-	MaxPageSize                 *int    `access:"authentication"`
+	SkipCertificateVerification *bool   `access:"authentication_ldap"`
+	PublicCertificateFile       *string `access:"authentication_ldap"`
+	PrivateKeyFile              *string `access:"authentication_ldap"`
+	QueryTimeout                *int    `access:"authentication_ldap"`
+	MaxPageSize                 *int    `access:"authentication_ldap"`
 
 	// Customization
-	LoginFieldName *string `access:"authentication"`
+	LoginFieldName *string `access:"authentication_ldap"`
 
-	LoginButtonColor       *string `access:"authentication"`
-	LoginButtonBorderColor *string `access:"authentication"`
-	LoginButtonTextColor   *string `access:"authentication"`
+	LoginButtonColor       *string `access:"authentication_ldap"`
+	LoginButtonBorderColor *string `access:"authentication_ldap"`
+	LoginButtonTextColor   *string `access:"authentication_ldap"`
```

Comment:
Hey Ben,

I wanted to make a quick note, so in `SectionsToSubSections` we noticed these actually belong to experimental features. When you merge `SectionsToSubSections` and resolve conflicts, make sure you keep them as the values they are currently. They should be the following

```
	LoginButtonColor       *string `access:"experimental_features"`
	LoginButtonBorderColor *string `access:"experimental_features"`
	LoginButtonTextColor   *string `access:"experimental_features"`
```